src/index.html

```diff
@@ -4,10 +4,320 @@
+              alt="Leslie Mckinney photo"
+              class="people-card__photo"
+            >
+            <h3>
+              Leslie Mckinney
+            </h3>
+            <p class="people-card__profession">
+              CEO
+            </p>
+            <p>
+              Nice work, Certificates National
+              Association of Realtors (USA)
+            </p>
+          </div>
+          <div class="people-card">
+            <img
+              src="./images/people/scarlett.png"
+              alt="Scarlett Hawkins photo"
+              class="people-card__photo"
+            >
+            <h3>
+              Scarlett Hawkins
+            </h3>
+            <p class="people-card__profession">
+              Senior Vice President
+            </p>
+            <p>
+              My soul is illuminated by an unearthly joy,
+              as these beautiful spring morning,
+              which I enjoy with all my heart
+            </p>
+          </div>
+          <div class="people-card">
+            <img
+              src="./images/people/max.png"
+              alt="Max Henry photo"
+              class="people-card__photo"
+            >
+            <h3>
+              Max Henry
+            </h3>
+            <p class="people-card__profession">
+              Chief Design Officer
+            </p>
+            <p>
+              When my lovely valley of steam rises
+              and half-day sun is above an impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="contacts content__contacts" id="contacts">
+        <div class="section-title contacts__section-title">
+          <h2>
+            Contacts
+          </h2>
+          <span class="section-title__icon"></span>
+        </div>
+        <div class="contacts__container">
+          <div class="contacts-info">
+            <div class="contacts-info__item">
+              <a href="tel:+1 234 555-55-55" class="contacts-info__link">
+                <span class="contacts-info__link-name">
+                  Phone
+                </span>
+                +1 234 5555-55-55
+              </a>
+              <a href="mailto:hello@miami.com" class="contacts-info__link">
+                <span class="contacts-info__link-name">
+                  Email
+                </span>
+                hello@miami.com
+              </a>
+            </div>
+            <a href="https://www.google.com/maps/place/400+N+1st+Ave+STE+700,+Minneapolis,+MN+55401,+%D0%A1%D0%A8%D0%90/@44.9813258,-93.2753198,17z/data=!3m1!4b1!4m5!3m4!1s0x52b3328fdf233887:0x3348e5113ed2472d!8m2!3d44.9813258!4d-93.2731311" class="contacts-info__link">
+              <span class="contacts-info__link-name">
+                Adress
+              </span>
+              400 first ave. n. <br> suite 700 <br>
+              Minneapolis, MN 55401
+            </a>
+          </div>
+          <form
+            action=""
```

Comment:
https://stackoverflow.com/questions/1131781/is-it-a-good-practice-to-use-an-empty-url-for-a-html-forms-action-attribute-a